web/app/models.py

```diff
@@ -225,9 +228,10 @@ def __str__(self):
             self.submitted)
 
 class TournamentPlayer(db.Model):
-    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    __tablename__ = 'tournament_player'
+    # id = db.Column(db.Integer, primary_key=True, autoincrement=True)
```

Comment:
I disagree with steve here -- Each new row should have a separate key distinct from any other ID. Imagine if we had tournaments in other countries.

It could conceivable be the key from the User table, but for now (and in general, I think it best practice) to have each table have its own unique key.